test/unit/component/parser/json/BpmnJsonParser.event.start.timer.test.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ShapeBpmnElementKind } from '../../../../../src/model/bpmn/shape/ShapeBpmnElementKind';
+import { parseJsonAndExpectOnlyEvent, parseJsonAndExpectOnlyFlowNodes, verifyShape } from './JsonTestUtils';
+import { ShapeBpmnEventKind } from '../../../../../src/model/bpmn/shape/ShapeBpmnEventKind';
+
+describe('parse bpmn as json for timer start event', () => {
+  it('json containing one process with an timer start event, timer start event is present', () => {
+    const json = `{
+                "definitions" : {
+                    "process": {
+                        "startEvent": {
+                            "id":"event_id_0",
+                            "name":"event name",
+                            "timerEventDefinition": ""
```

Comment:
Missing the same test, where `"timerEventDefinition": {id:""}`

Reply:
@csouchet I have manage this with 8886515
Notice that terminate end event doesn't have such a test

Reply:
@tbouffard It's weird. It seems to me to add this test. Maybe a bad remove :(

Reply:
The remove of the test :slightly_frowning_face: : 
https://github.com/process-analytics/bpmn-visualization-js/pull/170/commits/dba19a86a3f7e123deadb10c26faf1605fd10347#diff-63918928c1583e55e5d1048e592ca76e

Reply:
this is probably when we detect that the event definition can be an empty string or an object, and we only kept the string form. That's also why I forgot the object form test at the beginning.
Anyway, that is not a big deal: we can add the missing tests, new event types will be covered and as the parsing implementation is the same, the code is already covered by tests